src/modules/project/pages1/projectDashboard/common/TableWithAccordion.tsx

```diff
@@ -169,15 +174,17 @@ export function TableItemWithAccordion<TItem>({
               textAlign={row.key === 'dropdown' ? 'right' : 'left'}
             >
               {row.key === 'dropdown' ? (
-                <IconButton
-                  size="sm"
-                  alignSelf={'center'}
-                  aria-label="dropdown"
-                  variant="ghost"
-                  colorScheme="neutral1"
-                  icon={isOpen ? <PiCaretUp fontSize={'16px'} /> : <PiCaretDown fontSize={'16px'} />}
-                  onClick={onToggle}
-                />
+                showAccordion ? (
+                  <IconButton
+                    size="sm"
+                    alignSelf={'center'}
+                    aria-label="dropdown"
+                    variant="ghost"
+                    colorScheme="neutral1"
+                    icon={isOpen ? <PiCaretUp fontSize={'16px'} /> : <PiCaretDown fontSize={'16px'} />}
+                    onClick={onToggle}
+                  />
+                ) : null
```

Comment:
This should be placed outside of the return at 160 and set to return null.

if(row.key === 'dropdown' && !showAccordion) return null.

I remember we did this with AI, and it was showing blank. It's because there's still the TD, component that has null in it. If there's no TD component, then the blank area would be gone.